cg/meta/orders/utils.py

```diff
@@ -26,3 +27,21 @@ def get_ticket_tags(order: Order, order_type: OrderType) -> list[str]:
             tags.append("existing-data")
 
     return tags
+
+
+def contains_only_existing_samples(order: OrderWithCases) -> bool:
+    """Check if the order contains only existing samples"""
+
+    for enumerated_case in order.enumerated_cases:
+        case: Case = enumerated_case[1]
+        if case.enumerated_new_samples:
+            return False
+    return True
+
+
+def get_ticket_status(order: Order) -> int:
```

Comment:
Is this not what is returned?
```suggestion
def get_ticket_status(order: Order) -> Status:
```